src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -1040,7 +1040,7 @@
           short-summary: The OS Type. Linux or Windows.
         - name: --os-sku
           type: string
-          short-summary: The os-sku of the agent node pool. Ubuntu or CBLMariner.
+          short-summary: The os-sku of the agent node pool. Ubuntu or CBLMariner when os-type is Linux, default is Ubuntu if not set; Windows2019 or Windows2022 when os-type is Windows, default is Windows2019 if not set.
```

Comment:
@ShiqianTao Does cli code set the default value for Windows? If it is set by rp, we may change the default value in future so we need to mention it in the description.

Reply:
Yes, AKS-RP controls the default value. Updated the description.